lib/msal-common/src/error/ClientConfigurationError.ts

```diff
@@ -47,7 +47,7 @@ export const ClientConfigurationErrorMessage = {
     },
     invalidPrompt: {
         code: "invalid_prompt_value",
-        desc: "Supported prompt values are 'login', 'select_account', 'consent' and 'none'.  Please see here for valid configuration options: https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-js-initializing-client-applications#configuration-options",
+        desc: "Supported prompt values are 'login', 'select_account', 'consent', 'create' and 'none'.  Please see here for valid configuration options: https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-js-initializing-client-applications#configuration-options",
```

Comment:
The link here maybe a bit redundant? If we want to keep it can we change it to the typedocs link instead as I think the docs.mso link is a bit outdated?

Reply:
And I would say it should be an aka.ms link, in case we want to change it.

Reply:
(and also shouldnt have `en-us`)

Reply:
Changed this to the typedoc link.